### ISSUE TITLE
C in 6pt wght>=500 

### ISSUE THREAD

**Issue by MariannaPaszkowska** [googlefonts/googlesans-flex#240]
`C` feels too heavy at opsz = 6 in the wght>=500 area of the weight range. This is visible at 9-12 pt too. 
Please review.

**Comment by dberlow** [comment]
I am not seeing this in chrome or safari at opsz6-wdth100-wght500-GRAD0-ROND0, or opsz9-12.

Safari
![1F812A95-250B-4522-915A-35E009C1D773](https://user-images.githubusercontent.com/1956494/212403412-a2afb999-bc7c-4836-98ba-461f9ea58da0.jpeg)


Chrome ![image](https://user-images.githubusercontent.com/1956494/212402619-fb20be04-6adb-41b9-ab06-fbba56ae897a.jpeg)

**Comment by dberlow** [comment]
I needed to get back to open the sources around this style and make sure there was an issue, which is what needs to be done after looking at the resulting type and seeing nothing wrong. 

I then generated the 6 pt 500 to see is anything was happening unexpectedly, and it is not. They are pixel perfect for that glyph at the location in the designspace. 
<img width="28" alt="Screen Shot 2023-01-17 at 8 45 22 AM" src="https://user-images.githubusercontent.com/1956494/212915154-36d155bb-a64d-4319-bbc1-f8e91ec66af9.png">
